src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/messages.svg

```diff
@@ -0,0 +1,10 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="24" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
```

Comment:
If it's an icon from another project, it need to be credited. And we need to be sure the license is compatible with the MIT.

Reply:
I will replace this one with the one from Material Icons projects which is compatible 👍

Reply:
Don't worry. When this is merged, we'l ask Symfony's designer to create the icon. He created all the other ones, so it's important to do that to keep icons consistent.